spec/features/menu_spec.rb

```diff
@@ -15,7 +15,6 @@
 
     within ".navbar" do
       expect(page).to have_content("Home")
-      expect(page).to have_content("Initiatives")
```

Comment:
This has been removed as https://github.com/decidim/decidim/pull/9928 was introduced in this release. The logic changes, so if there isn't any InitiativeType, the Initiatives menu isn't show (as anybody can't create any Initiative).